src/Behat/Behat/Transformation/Transformation/ReturnTypeTransformation.php

```diff
@@ -130,7 +130,7 @@ static private function getReturnClass(ReflectionFunctionAbstract $reflection)
             return null;
         }
 
-        return (string) $type;
+        return $type->getName();
```

Comment:
This will brake compatibility with php 5.3

Reply:
Ok, I changed it